Makefile

```diff
@@ -19,23 +20,36 @@ FULL    := $(shell bin/get_version.sh --full)
 
 # Set this variable during 'make install' to specify the Perl interpreter used in
 # installed scripts, or leave empty to keep the current interpreter.
-export LCOV_PERL_PATH := /usr/bin/perl
+export LCOV_PERL_PATH := /usr/bin/env perl
```

Comment:
I don't think we want to change this for the default case - the default should be that anything installed in /usr/bin specifies exactly how it needs to be run, including the full path to the associated script interpreter. Having /usr/bin/env in the mix for these installed files will change the behavior of global binaries depending on the path setting of each user. If there's a need to use env for installed files, a user can simply specify LCOV_PERL_PATH as part of the make install invocation.

Reply:
Either way is fine.
The main issue for me (and I suspect, for many other companies) is that some systems are pretty old, and their /usr/bin tools can be archaic.  As a result, we very strongly recommend everyone to write scripts with /usr/bin/env.
But - a workaround exists, so this is not a major concern in this case.